### PR TITLE
chore: added commercial API to  LICENSE

### PR DESCRIPTION
**Pull request by PeerRich**

### CODE REVIEW

**Review by pumfleet — DISMISSED** [review]
Should be the tree URL: https://github.com/calcom/cal.com/tree/main/apps/api

**Comment by xcskier56** [comment]
Hi there! Commenting here b/c very relevant. I'm in the middle of evaluating Cal.com for use in our product and the API is a key component. My understanding was that the API was intended to be open-source licensed as referenced here: https://cal.com/docs/introduction/quick-start/self-hosting/installation/platform/api-submodule.

From the looks of things, you're in the middle of transitioning this code's license. Are you able to clarify what the license status of the API module is, and what it will be going forward? 

Thank you -- Doug

File: LICENSE [hunk 1]
```diff
@@ -2,7 +2,7 @@ Copyright (c) 2020-present Cal.com, Inc.
 
 Portions of this software are licensed as follows:
 
-* All content that resides under https://github.com/calcom/cal.com/tree/main/packages/features/ee directory of this repository (Commercial License) is licensed under the license defined in "ee/LICENSE".
+* All content that resides under https://github.com/calcom/cal.com/tree/main/packages/features/ee and https://github.com/calcom/cal.com/blob/main/apps/api/ directory of this repository (Commercial License) is licensed under the license defined in "ee/LICENSE".
```
Comment (PeerRich) [inline LICENSE:5]:
```suggestion
* All content that resides under https://github.com/calcom/cal.com/tree/main/packages/features/ee and https://github.com/calcom/cal.com/tree/main/apps/api/ directory of this repository (Commercial License) is licensed under the license defined in "ee/LICENSE".
```

**Review by sean-brydon — APPROVED** [review]

**Comment by xcskier56** [comment]
@PeerRich, thank you for the clarification.

**Pull request merged**